tests/functional/test_rose_stem.py

```diff
@@ -18,60 +18,16 @@
 # along with Rose. If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------
 
-"""
-Tests for cylc.rose.stem
-========================
-
-These tests are based on the Rose 2019 tests for Rose Stem.
-
-Structure
----------
-
-#. ``setup_stem_repo`` is a module scoped fixture which creates a Rose Stem
-   repository which is used for all the tests.
```

Comment:
These docs were broken by previous changes.

The relevant info should be in the docstrings of the fixtures / methods.

Reply:
This whole structure was nasty. I'm glad you've killed it.

My aim at the time I did it was to replicate the old functional tests as closely as possible without adding the TAP framework to Cylc Rose.